askama/src/values.rs

```diff
@@ -7,11 +7,15 @@ use crate::Error;
 pub const NO_VALUES: &dyn Values = &();
 
 /// Try to find `key` in `values` and then to convert it to `T`.
+#[inline]
 pub fn get_value<T: Any>(values: &dyn Values, key: impl AsRef<str>) -> Result<&T, Error> {
-    let Some(src) = values.get_value(key.as_ref()) else {
-        return Err(Error::ValueMissing);
-    };
+    values
+        .get_value(key.as_ref())
+        .ok_or(Error::ValueMissing)
+        .and_then(convert_value)
+}
 
+fn convert_value<T: Any>(src: &dyn Any) -> Result<&T, Error> {
```

Comment:
What's the advantage of creating this function btw?

Reply:
The generated byte code of this function is huge, so I would not want to inline it.

Reply:
:+1: